### ISSUE TITLE
Metalsmith · metalsmith.join is not a function

### ISSUE THREAD

**Issue by corzani** [segmentio/metalsmith#186]
Node: 5.0.0
Npm: 3.3.9
Metalsmith: 2.1.0

I've tried Metalsmith using the Yeoman generator (https://www.npmjs.com/package/generator-metalsmith).
This is the output I receive:

```

Malevich:test user$ metalsmith

Metalsmith · metalsmith.join is not a function

TypeError: metalsmith.join is not a function
    at convert (/Users/user/test/node_modules/metalsmith-templates/lib/index.js:78:26)
    at /Users/user/test/node_modules/metalsmith-templates/node_modules/async/lib/async.js:111:13
    at Array.forEach (native)
    at _each (/Users/user/test/node_modules/metalsmith-templates/node_modules/async/lib/async.js:32:24)
    at async.each (/Users/user/test/node_modules/metalsmith-templates/node_modules/async/lib/async.js:110:9)
    at Ware.<anonymous> (/Users/user/test/node_modules/metalsmith-templates/lib/index.js:64:5)
    at Ware.<anonymous> (/usr/local/lib/node_modules/metalsmith/node_modules/wrap-fn/index.js:45:19)
    at next (/usr/local/lib/node_modules/metalsmith/node_modules/ware/lib/index.js:85:20)
    at /usr/local/lib/node_modules/metalsmith/node_modules/wrap-fn/index.js:121:18
    at Ware.<anonymous> (/Users/user/test/node_modules/metalsmith-collections/lib/index.js:116:5)

```

**Comment by Ajedi32** [comment]
So... did you figure out what the problem is? I'm assuming you did since you closed this issue?

**Comment by corzani** [comment]
By running the metalsmith command directly was not working, the output was the previous one.
I found the issue, there is a Makefile and so I've checked and executed it, everything was working fine.
The makefile is using the local copy of metalsmith ("node_modules/.bin/metalsmith") so the 0.10.0 version specified in package.json and I was using the 2.10 installed with "npm install -g metalsmith".
I suspect that was because I didn't have all the dependencies I needed using the global metalsmith.
So... my fault...
